core/static/js/ga.js

```diff
@@ -0,0 +1 @@
+// override this file in your theme to add your own google analytics code
```

Comment:
We will have some block override examples in Karin's templates branch, but I wonder if having a whole different google analytics file would be better than a block hidden inside a template?